### ISSUE TITLE
Plugin doesn't show up automatically in Napari after normal installation

### ISSUE THREAD

**Issue by Theresa-S** [rosalindfranklininstitute/RedLionfish#15]
I followed the installation instructions for conda and put lionfish into the same virtual environment as napari (fresh version). 
![grafik](https://user-images.githubusercontent.com/61883896/208865344-58032b0d-a42e-4dbd-93d5-e6f7bb853241.png)

It now appears in the folder of the environment:
![grafik](https://user-images.githubusercontent.com/61883896/208865057-b30be4f4-6c40-41ea-bf31-422bbd7c28f8.png)

However, the plugin is not recognized as installed:
![grafik](https://user-images.githubusercontent.com/61883896/208864874-09aacfc4-d8c7-48cd-84e1-300c0d5229a5.png)

Is this a bug or did I miss something? Disclosure: I have no idea about coding ;-)
(and yes, the environment was activated before calling napari)

**Comment by Theresa-S** [comment]
PS: installing redlionfish via the "Install Plugins" function works. However, the plugin itself doesn't work:

(napari-env1) C:\Users\there\miniconda3>napari
iterations = 10
data.shape = (60, 1026, 1544) , type(data) = <class 'numpy.ndarray'> , data.dtype = uint8
psfdata.shape = (61, 32, 32) , type(psfdata) = <class 'numpy.ndarray'> , psfdata.dtype = uint8
C:\Users\there\miniconda3\lib\site-packages\reikna\cluda\dtypes.py:187: FutureWarning: In the future `np.bool` will be defined as the corresponding NumPy scalar.  (This may have returned Python scalars in past versions.!
10:08:42 ERROR Failed to setup Reikna with OpenCL.
10:08:42 ERROR module 'numpy' has no attribute 'bool'

I got the same error message when using an environment with Napari installed via devbio-napari.

**Comment by perdigao1** [comment]
Hi Theresa,

Thank you for trying and reporting the error. I am not sure why you are getting this error. I will try on my PC. My suspicion is that you may be using a later version of python and numpy, or it could be that the package Reikna has been updated and has a new issue. If you could report back the versions of these packages (python, numpy, reikna, redlionfish) it would help. Thank you.

**Comment by perdigao1** [comment]
Further research, I managed to reproduce this error. This problem seems to come from the fact that numpy changed the way it deals with numpy.bool types. There is a warning but, in fact, it now throws error when trying to use it. Reikna tries to register this type, even though it is not used by redlionfish.

As a temporary solution, I suggest downgrading numpy from 1.24 to version 1.23.2, `conda install numpy=1.23.2`

I filed this issue at
[https://github.com/fjarri/reikna/issues/63](https://github.com/fjarri/reikna/issues/63)
